include/mbedtls/bignum.h

```diff
@@ -129,6 +129,7 @@
         #endif /* !MBEDTLS_HAVE_INT64 */
 typedef  int64_t mbedtls_mpi_sint;
 typedef uint64_t mbedtls_mpi_uint;
+#define MBEDTLS_MPI_UINT_MAX                18446744073709551615UL
```

Comment:
18446744073709551615 is not guaranteed to fit in a `long`, so having the `L` suffix is misleading. This should be either
```suggestion
#define MBEDTLS_MPI_UINT_MAX                18446744073709551615U
```
(which is perfectly correct, the compiler will choose the smallest type in the hierarchy where the value fits) or
```suggestion
#define MBEDTLS_MPI_UINT_MAX                18446744073709551615ULL
```
(which may select an overly large type on a platform where `unsigned long long` is more than 64 bits, but that's extremely uncommon).